src/main/java/org/apache/mesos/curator/CuratorStateStore.java

```diff
@@ -373,6 +382,32 @@ public void clearProperty(final String key) throws StateStoreException {
         }
     }
 
+    @Override
+    public List<Protos.Resource> getExpectedResources() {
```

Comment:
Given that this is effectively a utility method on top of fetchTasks(), maybe it would fit better in a separate utility that's upstream of/separate from the StateStore itself?